app/helpers/helper.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace App\Helpers;
+
+class Helper
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR1.Methods.CamelCapsMethodName.NotCamelCaps</br>**Severity**: error</br>**File**: app/helpers/helper.php</br>**Line**: 6</br>Method name "Helper::index_number" is not in camel caps format